x-pack/snapshot-tool/build.gradle

```diff
@@ -82,6 +82,19 @@ dependencyLicenses {
 test.enabled = false
 testingConventions.enabled = false
 
+// We make AbstractCleanupTests class available to qa submodules by creating testArtifacts configuration
```

Comment:
Again, let's please put a comment here to replace this with the test-fixtures plugin that is coming in Gradle 5.6.